packages/ember-glimmer/lib/utils/bindings.js

```diff
@@ -14,6 +14,17 @@ function referenceForKey(component, key) {
 }
 
 function referenceForParts(component, parts) {
+  let isAttrs = parts[0] === 'attrs';
```

Comment:
Upstream from this [the check is done](https://github.com/emberjs/ember.js/blob/473a4f5f05423837c70376d8c12a5206aa2f8629/packages/ember-glimmer/lib/utils/bindings.js#L66-L67) to make sure we have a path reference.